src/ar.erl

```diff
@@ -405,11 +415,19 @@ generate_logfile_name() ->
 	{{Yr, Mo, Da}, {Hr, Mi, Se}} = erlang:universaltime(),
 	lists:flatten(
 		io_lib:format(
-			"~s/session_~4..0b-~2..0b-~2..0b_~2..0b-~2..0b-~2..0b.log",
-			[?LOG_DIR, Yr, Mo, Da, Hr, Mi, Se]
+			"~s/session_~4..0b-~2..0b-~2..0b_~2..0b-~2..0b-~2..0b~s.log",
+			[?LOG_DIR, Yr, Mo, Da, Hr, Mi, Se, maybe_node()]
 		)
 	).
 
+maybe_node() ->
+	case node() of
```

Comment:
What do you think about getting the sname from `init:get_argument(sname)` instead?

Reply:
✅